start.ps1

```diff
@@ -109,7 +148,10 @@ function main {
     } elseif ($FunctionName -eq "setenv") {
         Change-Cwd
         set_env
-    } else {
+    } elseif ($FunctionName -eq "getdependencies") {
```

Comment:
Could this be also added into tool's `Default-Func` please.

Reply:
here is what I would add 

```
function Default-Func {
    Write-Host ""
    Write-Host "LabLib library"
    Write-Host ""
    Write-Host "Usage: ./start.ps1 [target]"
    Write-Host ""
    Write-Host "Runtime targets:"
    Write-Host "  install                       Install Poetry and update venv by lock file."
    Write-Host "  setenv                        Set all env vars in .env file."
    Write-Host "  getdependencies               Download and extract all dependencies into vendor folder."
    Write-Host ""
}
```

Reply:
and also since you will be there how about to change **[target]** to **commands** (that was probably some residual code from other tool)

Reply:
sure thing! will do so swiftly

Reply:
addressed in 034e6d0e0554c71eba83512767c7dd34ccadaed7